habitat/sims/habitat_simulator.py

```diff
@@ -34,10 +34,10 @@ def check_sim_obs(obs, sensor):
 
 
 class SimulatorActions(Enum):
-    LEFT = "look_left"
-    RIGHT = "look_right"
-    FORWARD = "move_forward"
-    STOP = "stop"
+    FORWARD = 0
+    LEFT = 1
+    RIGHT = 2
+    STOP = 3
```

Comment:
Do we want to use this moment and make STOP=0?

Reply:
That should stay in #53 IMO.  Clearer commit history